dist/tools/zep_dispatch/README.md

```diff
@@ -144,3 +144,20 @@ instance [0 | Iface: 7 | mop: 2 | ocp: 0 | mhri: 256 | mri 0]
 ```
 
 This should also be visible in Foren6.
+
+Topology generation
+-------------------
+
+To generate a random topology use the `topogen.sh` script.
+This will randomly distribute *N* nodes on on a *W* × *H* map.
+Each node has a radio range *R* ± *V* where *V* is a random variance that can also be set to 0.
+
+The further away a node is from a sending node, the higher the packet loss probability.
+Nodes outside the sending radius have a zero probability of receiving a packet.
```

Comment:
Do you know if it is possible to draw an ellipse filled with a color gradient instead of a solid color in gnuplot?

Reply:
Couldn't find a way quickly, seems it needs higher level gnuplot skills to get this. Matplotlib can do this, but that means rewriting the code.